cs/src/core/Index/Common/AddressInfo.cs

```diff
@@ -74,7 +74,7 @@ public long Size
 
         public long Address
         {
-            readonly get
+            get
```

Comment:
why is readonly marker removed?

Reply:
Remnant from embedding source code to net462 - will re-add this back.